util/xs_scripts/rvv_vlen/test_elem_gen_idx_negative.py

```
@@ -0,0 +1,84 @@
+class TestNegativeElemGenIdx(unittest.TestCase):
+    def test_correct_path_passes_all_vlens(self):
+        for vlen in (128, 256, 512):
+            out0, out1 = simulate_vlseg2e8(vlen, True)
+            vl = len(out0)
+            expect0 = [0xA0 + 2 * i for i in range(vl)]
+            expect1 = [0xA0 + 2 * i + 1 for i in range(vl)]
+            self.assertEqual(out0, expect0, f"vlen={vlen}")
+            self.assertEqual(out1, expect1, f"vlen={vlen}")
+
+    def test_default128_fails_when_arch_is_256(self):
+        out0, out1 = simulate_vlseg2e8(256, False)
```

Ruff (0.16.1)

[warning] Unpacked variable `out1` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)